<?xml version="1.0"?>
<launch>

	<node pkg="multi_localizer" type="sampler" name="sampler" output="screen" >
		<param name="SAMPLE_NUM"  value="10" />
		<param name="FILE_PATH"   value="$(find multi_localizer)/record/20220630/" />
		<param name="RECORD_PATH" value="$(find multi_localizer)/record/" />
	</node>

</launch>